src/locales/en/name/sex.ts

```
@@ -0,0 +1 @@
+export default ['female', 'male', 'non-binary'];
```

IMO this is a wrong change :thinking: 
Originally this was binary_gender file, which contains translated strings which could also be used in `faker.helpers.fake` (which can be used for e.g. stored database strings)
Also now the values are in lower case as before they were in normal case starting with an upper case letter
Beside all of that, if we name this sex it could get into confusion with the sex enum / sex property of functions, but this would not work by design as this locale definitions are translated values and e.g. in locale fr it is called `Femme` and `Homme`, so they have a totally different technical meaning as the property/enum.

---

This continues to be stored in the dictionary, just `sex` instead of `binary_gender`. This can continue to be used as it was, just in a better way, as it no includes 3 items instead of 2, so it does express better sex selection on different forms during the registration process - more and more websites offer you to specify "non-binary" as an option. So if anything, we improved the data.

There is no reason why this data should be written in capital form, as this is just a pure noun. So why do you think it should be written in capital form? Other nouns we always write in small letters...

I believe we specify `sex` parameter as `SexType` so there should be no confusion when used via typescript. When used in plain JS, we should probably throw when an unsupported value is provided, which would solve the problem...

The naming of the function does not make any difference in this proposal vs the current situation, as even currently people were already trying to misuse it. So we should solve it on the code level and not by naming the funct this or the other way...